app.js

```diff
@@ -0,0 +1,63 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const form = document.getElementById('resume-form')
+  const schema = document.getElementById('schema')
+  const addProfile = document.getElementById('add-profile')
+  
+  form.addEventListener('submit', e => {
+    e.preventDefault()
+    const fd = new FormData(form)
```

Comment:
Having done mostly React, vanilla JS is really interesting. Will have to read more extensively on this. https://javascript.info/formdata